core/vm/interpreter.go

```diff
@@ -35,6 +35,8 @@ type Config struct {
 
 	StatelessSelfValidation bool // Generate execution witnesses and self-check against them (testing purpose)
 	EnableWitnessStats      bool // Whether trie access statistics collection is enabled
+
+	ExposeMultiGas bool // Arbitrum: Expose multi-gas used in transaction receipts
```

Comment:
Is there somewhere to ensure that it defaults to `false`? Ideally, somewhere that we can later make it default to `true` in some future nitro node release.